proto/ibc/core/client/v1/tx.proto

```
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package ibc.core.client.v1;
+
+option go_package = "github.com/cometbft/cometbft/proto/ibc/core/client/v1";
+
+import "gogoproto/gogo.proto";
```

buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)